src/subsys/ddebug.h.tt2

```diff
@@ -56,21 +56,21 @@ dd(const char *fmt, ...) {
 
 #define dd_check_read_event_handler(r)                                       \
     dd("r->read_event_handler = %s",                                         \
-        r->read_event_handler == ngx_http_block_reading ?                    \
```

Comment:
I think we should just keep those as-is since they came directly from https://github.com/openresty/lua-nginx-module/blob/master/src/ddebug.h?

Reply:
Right, these changes are not really desired.

Reply:
@dndx @agentzh 
`mini-ttl.pl` will check template's function call indentation before it generates the target.
The origin code is not well indented.
If we need to apply the function call indentation to macro, we need to fix the template first.

Reply:
@spacewander I suggest you leave this macro alone. It does not matter since it is always aligned up in the C output anyway.

Reply:
@agentzh 
It doesn't matter whether the C output.
1. `mini-ttl.pl` will check the function call indentation before generating C output.
2. After we apply the indentation to macro, `mini-ttl.pl` will check the indentation in macro too.
3. The macro template is not well indented.
4. So it could not pass the indentation check.

Without indenting this snippet, we will get error message like this:
```
src/subsys/ddebug.h.tt2: line 59: continued arguments not aligned with the function call on line 58.
Makefile:27: recipe for target '/tmp/ddebug.h' failed
```

Reply:
I think @agentzh meant to keep both the template and output files well indented. Not just the output.

Reply:
@dndx 
Yes, it is also what I mean. Note that the origin template is not well indented, my change indents it.

Reply:
@spacewander I think the original indentation here is much better and (much more readable) than the new form. Please revert these changes in the `src/subsys/ddebug.h.tt2` template file.

Reply:
@agentzh 
If we need to apply the indentation check in macro, the original indentation could not pass.
```
src/subsys/ddebug.h.tt2: line 59: continued arguments not aligned with the function call on line 58.
Makefile:27: recipe for target '../lua-nginx-module/src/ddebug.h' faile
```

Reply:
@spacewander OK, I see. The check is quite strict...it does not yet take into account extra indentation required by `? :` ternary expressions.